etcdctl/ctlv3/command/elect_command.go

```diff
@@ -37,6 +39,7 @@ func NewElectCommand() *cobra.Command {
 		Run:   electCommandFunc,
 	}
 	cmd.Flags().BoolVarP(&electListen, "listen", "l", false, "observation mode")
+	cmd.Flags().BoolVarP(&electDebug, "debug", "d", false, "debug mode")
```

Comment:
I will rework on this tonight. The reason why I put this flag is to get the pid from the process printed as explained in https://github.com/coreos/etcd/pull/5014#issuecomment-208437155. Not sure why `Process.Pid` returns the `ppid` of the process. Thanks!